static/js/settings_org.js

```diff
@@ -276,12 +276,16 @@ exports.populate_realm_domains = function (realm_domains) {
         realm_domain => realm_domain.allow_subdomains ? "*." + realm_domain.domain : realm_domain.domain
     );
     let domains = domains_list.join(', ');
+    const zero_domains_warning = $ ("#zero_domains_warning");
     if (domains.length === 0) {
+        zero_domains_warning.text(i18n.t("There are no domains configured now. This'd restrict anyone from registering."));
+        zero_domains_warning.show();
         domains = i18n.t("None");
+    } else {
+        zero_domains_warning.hide();
     }
-    $("#allowed_domains_label").text(i18n.t("Allowed domains: __domains__", {domains: domains}));
 
-    const realm_domains_table_body = $("#realm_domains_table tbody").expectOne();
+    const realm_domains_table_body = $("#realm_domains_table tbody");
```

Comment:
Made this change because when a normal user logins `#realm_domains_table` is not expected to be present.

Reply:
Good catch! We should have a better fix than this, as this removes a sanity check from the code. Maybe `if(!is_admin) return;` somewhere above this line.

Reply:
Added that `if` in the top like this because we do not  want to populate realm domains table for normal users
```
exports.populate_realm_domains = function (realm_domains) {
    if (!meta.loaded || !page_params.is_admin) {
        return;
    }